libraries/AP_HAL_ChibiOS/RCOutput.cpp

```diff
@@ -2232,7 +2239,7 @@ void RCOutput::_set_profiled_clock(pwm_group *grp, uint8_t idx, uint8_t led)
     const uint8_t bit_length = 25;
     const uint8_t stride = 4;
     uint32_t *buf = grp->dma_buffer + (led * bit_length + pad_start_bits) * stride + idx;
-    const uint32_t BIT_1 = 7 * grp->bit_width_mul;
+    const uint32_t BIT_1 = NEOP_BIT_0_TICKS * grp->bit_width_mul;
```

Comment:
Maybe a different define for ProfiLEDs.

Reply:
@IamPete1 sure! Are you able to point me at a spec where it specifies the pulse widths?

Reply:
No, its not actually a pulse width based protocol. Its is clock and data. It uses the different pulse widths to delay the clock rising edge relative to the data one. It should be quite robust to timing due to this. Just don't want someone to change the neopixel widths and accidentally break ProfiLEDs. In theory we should also be able to significantly increase the clock speed too, but it didn't seem worth pushing it too far.

Reply:
Fixed